spec/presenters/firm_profile_presenter_spec.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.describe FirmProfilePresenter do
+  subject do
+    described_class.new(firm_result)
+  end
+
+  let(:firm_result) { double(FirmResult) }
+
+  describe '#type_of_advice_method' do
+    context 'face to face available' do
+      it do
+        expect(firm_result).to receive(:in_person_advice_methods).and_return([1, 2, 3])
+        expect(subject.type_of_advice_method).to eq(:face_to_face)
+      end
+    end
+
+    context 'face to face not available' do
+      it do
```

Comment:
Same thing here re spec explanation text of desired outcome. Or do you feel that the actual steps does a good job of describing this?